source/src/ble/cs_BleCentral.cpp

```diff
@@ -13,9 +13,10 @@
 #include <events/cs_EventDispatcher.h>
 #include <logging/cs_Logger.h>
 #include <storage/cs_State.h>
+#include <structs/buffer/cs_EncryptionBuffer.h>
 
 #define LOGBleCentralInfo LOGi
-#define LOGBleCentralDebug LOGnone
+#define LOGBleCentralDebug LOGv
```

Comment:
set back to none

Reply:
Ah, this is a result of me accidentally setting `#define LogLevelBleCentralDebug SERIAL_NONE` which makes it the highest level of logging, in contrast to LOGnone, which makes it never log anything. Maybe we should introduce a new SERIAL level, or rename LOGnone?

Reply:
Arend has a winner: we introduce LOGvv, LOGvvv, etc. This equals to SERIAL_VERY_VERBOSE, SERIAL_VERY_VERY_VERBOSE, etc.